web-components-f902/notes/04_javascript-web-components-explicit-api-s-using-class-methods.md.md

```diff
@@ -0,0 +1,41 @@
+# Web Components explicit API's using class methods
+
+[Video link](https://egghead.io/lessons/javascript-web-components-explicit-api-s-using-class-methods)
+
+<TimeStamp start="0:18" end="0:28">
+
+```jsx
```

Comment:
Are you using the code block notes so the user can conveniently copy?

Reply:
Sort of, this was establishing the baseline at the start of the lesson and was trying to find a way to help contribute to a viewer's experience. I realize now that if they're looking to copy code, they would have the actual lesson code pulled up, so this is likely worth removing.